lib/criticalcss.js

```diff
@@ -74,7 +74,7 @@ criticalCSS by @scottjehl && @jefflembeck. Run this on your CSS, get the styles
 		critical = critical.filter( noEmpties );
 
 		if( critical.length  === 0 ){ return ""; }
-		return critical.join( "\n" );
+		return critical;
```

Comment:
I'd watch out for this one. I've had it bite me before (I wish I remembered the context).